Tests/ADCTest.cpp

```diff
@@ -3,42 +3,46 @@
 #include "HALALMock/Services/SharedMemory/SharedMemory.hpp"
 #include <iostream>
 
+TEST(ADC, Check_Memory_Writing){
+
+    SharedMemory::start("GPIO_Name", "State_Machine_Name");
+
+    for (int i = 0; i < SharedMemory::total_pins; i++) {
+        ((uint8_t *)(SharedMemory::gpio_memory + i))[1]=1;
+        EXPECT_EQ(((uint8_t *)(SharedMemory::gpio_memory + i))[1],1);
+        ((uint8_t *)(SharedMemory::gpio_memory + i))[1]=0;
+        EXPECT_EQ(((uint8_t *)(SharedMemory::gpio_memory + i))[1],0);
```

Comment:
porqué `[1]` y no `[0]` ?  simplemente por curiosidad hehe :smiley:

Reply:
If i'm not wrong, before skipping the alignment, offset [0] was accesible, the problem was when we tried to access offset [1], because there was padding in between, so the first element of the struct was in offset [8]. Wrote that thinking that we could check if offset [1] was accessible after skipping the alignment, but i think it was a dumb thought, as those offsets were already accessible before but with no info stored, so yes, we could use any offset there.